robot/Cumulus/resources/ContactPageObject.py

```diff
@@ -64,8 +64,12 @@ def waitfor_actions_dropdown_and_click_option(self,option):
            Click on the desired option passed as a parameter
         """
         loc=npsp_lex_locators['contacts_actions_dropdown_menu']
+        self.npsp.click_more_actions_button()
         self.selenium.wait_until_element_is_visible(loc)
         self.selenium.click_link(option)
+        if option == 'Recalculating Rollups':
+            notification = npsp_lex_locators['crlps']['rollup_progress_notification'].format("rollups batch job is calculating")
+            self.selenium.wait_until_element_is_visible(notification)
```

Comment:
I dont see this being used anywhere, are these changes required?

Reply:
Removed it. Thought we would need it earlier but not using